plugins/nf-azure/src/main/nextflow/cloud/azure/config/AzStorageOpts.groovy

```diff
@@ -29,27 +29,27 @@ import nextflow.util.Duration
 @CompileStatic
 class AzStorageOpts {
 
-    private Map<String,String> sysEnv
+    private Map<String, String> sysEnv
     String accountKey
     String accountName
     String sasToken
     Duration tokenDuration
-    Map<String,AzFileShareOpts> fileShares
+    Map<String, AzFileShareOpts> fileShares
 
 
-    AzStorageOpts(Map config, Map<String,String> env=null) {
-        assert config!=null
-        this.sysEnv = env==null ? new HashMap<String,String>(System.getenv()) : env
+    AzStorageOpts(Map config, Map<String, String> env = null) {
+        assert config != null
+        this.sysEnv = env == null ? new HashMap<String, String>(System.getenv()) : env
         this.accountKey = config.accountKey ?: sysEnv.get('AZURE_STORAGE_ACCOUNT_KEY')
         this.accountName = config.accountName ?: sysEnv.get('AZURE_STORAGE_ACCOUNT_NAME')
         this.sasToken = config.sasToken
-        this.tokenDuration = (config.tokenDuration as Duration) ?: Duration.of('12h')
-        this.fileShares = parseFileShares(config.fileShares instanceof Map ? config.fileShares as Map<String,Map>
-                : Collections.<String,Map>emptyMap())
+        this.tokenDuration = (config.tokenDuration as Duration) ?: Duration.of('48h')
```

Comment:
Why it was increased to 2 days?

Reply:
This was done because in case the workflow goes beyond the previous default limit of `12h`, it is abruptly stopped and it affected some commercial users therefore I've increased it slightly.

Reply:
Fair enough